app/graphql/types/partner_type.rb

```diff
@@ -2,32 +2,57 @@ module Types
   class PartnerType < Types::BaseObject
     
 
-    description 'A Partner who runs Events'
+    description 'Organisations that run events'
 
-    # placecalID
-    field :id, ID, null: false
-    field :name, String, null: false
-    field :summary, String
-    field :description, String
+    field :id, ID, 
+      null: false,
+      description: 'ID of partner'
 
-    field :accessibility_summary, String, method: :accessibility_info
+    field :name, String, 
+      null: false,
+      description: 'A short string about this partner, an alias for `summary`'
 
-    field :logo, String
-    field :address, AddressType
+    field :summary, String,
+      description: 'A short string describing partner'
```

Comment:
A short summary of this partner

Reply:
are you saying the summary is a summary?